### ISSUE TITLE
Changes in the pushed module names?

### ISSUE THREAD

**Issue by kirill-grouchnikov** [vanniktech/gradle-maven-publish-plugin#191]
After the recent fixes for multiplatform modules in Kotlin 1.4.30 that went into version 0.14.0, I noticed a change in the structure and naming of the https://github.com/kirill-grouchnikov/aurora modules that were uploaded to Sonatype staging.

https://github.com/kirill-grouchnikov/aurora/tree/icicle/icon/icon used to go to https://oss.sonatype.org/content/repositories/snapshots/org/pushing-pixels/aurora-icon-icon/ but now is in https://oss.sonatype.org/content/repositories/snapshots/org/pushing-pixels/aurora-aurora-icon-icon-aurora-icon-icon/

Similar "changes" happened to the other Aurora modules. And what used to be

```
api("org.pushing-pixels:aurora-window:0.0.4-SNAPSHOT")
```

in the `dependencies` block of a sample app, now has to be

```
api("org.pushing-pixels:aurora-aurora-window:0.0.8-SNAPSHOT")
```

Is this intended?

**Comment by kirill-grouchnikov** [comment]
I'm still using `./gradlew uploadArchives` instead of `./gradlew publish` if that matters. Just muscle memory :)

**Comment by gabrielittner** [comment]
The task doesn't matter :)
It's not intended and I have an idea what could be causing it. I've changed how the plugin sets the artifact I'd because of a behavior change in Gradle. I think that the new way of doing it might cause this. I'll take a look

**Comment by gabrielittner** [comment]
My theory was correct, I will come up with a fix in the next days.

There is also a workaround you can do on your side. In your settings.gradle you can define this:
```
project(":icon:icon").setName("aurora-icon-icon")
project(":icon:transcoder").setName("aurora-icon-transcoder")
project(":component").setName("aurora-component")
project(":window").setName("aurora-window")
project(":skin").setName("aurora-skin")
project(":bitmapfilter").setName("aurora-bitmapfilter")
project(":common").setName("aurora-common")
```
and then remove all the `POM_ARTIFACT_ID` properties (you could also keep them but they won't do anything). That might actually the best way to set the artifact id in general, as it doesn't require us messing around with anything that might break. The only downside is that declaring dependencies on modules is a bit more verbose it's `implementation(project(":icon:aurora-icon-icon"))` instead of `implementation(project(":icon:icon"))`

**Comment by gabrielittner** [comment]
Just published 0.14.2 which should fix the issue

**Comment by kirill-grouchnikov** [comment]
Verified that the IDs / coordinates are back to what they were. Thanks!
